runtime/j9vm/javanextvmi.cpp

```diff
@@ -743,7 +743,7 @@ JVM_IsImplicitlyConstructibleClass(JNIEnv *env, jclass cls)
 JNIEXPORT jboolean JNICALL
 JVM_IsNullRestrictedArray(JNIEnv *env, jobject obj)
 {
-	assert(!"JVM_IsNullRestrictedArray unimplemented");
+	// TODO implement this when null restricted arrays have support
```

Comment:
Could you open an issue for the null restricted arrays support and put the issue number here.

Reply:
Actually there is an existing issue https://github.com/eclipse-openj9/openj9/issues/19460.

Reply:
I will add that to my comment